### PR TITLE
Make NDK faster than anything else

### PR DESCRIPTION
**Pull request by reyamir**
`tseep` is faster than `eventemitter3` by 13x, [source](https://github.com/Morglod/tseep/blob/master/benchmarks/results.md)

by replace `eventemitter3` with `tseep`, ndk is faster by 13x without doing anything 😎. I'm also tested it in Lume and it work totally fine, I didn't catch any issues so far

### CODE REVIEW

**Comment by Sebastix** [comment]
@reyamir Nice! Is there a way to apply these changes with a patch so I can test it? The @nostr-dev-kit/ndk package is build with turbo afaik.

**Comment by reyamir** [comment]
I'm not familar with turbo, but I've tested it by install ndk via local instead of git

my step:

1.  Clone project with branch `reyamir:feat/tseep`
2. Clone any project which using ndk
3. Replace ndk in this project by local ndk (clonned in step 1)

```
pnpm add <clonned git in step 1>
```

File: ndk/src/relay/pool/index.test.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import "websocket-polyfill";
```
Comment (erskingardner) [inline ndk/src/relay/pool/index.test.ts:1]: Why is this removed here as part of this PR?

Comment (reyamir) [reply ndk/src/relay/pool/index.test.ts:1]: oops, it just a mistake

Comment (erskingardner) [reply ndk/src/relay/pool/index.test.ts:1]: Want to give this a rebase while you're at it?

Comment (reyamir) [reply ndk/src/relay/pool/index.test.ts]: I've readd it

**Comment by pablof7z** [comment]
I've been using this and I think it's probably non-breaking; will play more tomorrow and if all good will merge

**Review by pablof7z — APPROVED** [review]

**Comment by Sebastix** [comment]
Just upgraded NDK from `2.0.5` to `2.1.1`

A very simple feed page which retrieves 50 events from 1 relay.
`2.0.5`:
https://shares.sebastix.dev/OQP1LqZu.png
`2.1.1`:
https://shares.sebastix.dev/jbmLg5HD.png

Yes, it loads faster :)
